frontend/src/app/auth/pages/signup/signup.page.ts

```diff
@@ -20,7 +20,7 @@ export class SignupPage implements OnInit {
               , private loadingController :LoadingController, private alertController : AlertController) {
     this.form = this.formBuilder.group(
       {
-        name: ['', [Validators.required, Validators.pattern('[a-zA-Z ]*')]],
+        name: ['', [Validators.required, Validators.pattern('[ a-zA-ZöçğüşÖÇĞÜŞİ]*')]], // turkish characters added öçğüşÖÇĞÜŞİ
```

Comment:
We should use a better validator that includes all of the non-english characters.